PowerPointLabs/PowerPointLabs/ELearningLab/Service/AudioSettingService.cs

```diff
@@ -1,11 +1,16 @@
 ﻿using System.Collections.Generic;
 using System.Collections.ObjectModel;
 using System.Linq;
+using Microsoft.Office.Tools;
+
+using PowerPointLabs.ActionFramework.Common.Extension;
 using PowerPointLabs.ELearningLab.AudioGenerator;
+using PowerPointLabs.ELearningLab.ELearningWorkspace.Views;
 using PowerPointLabs.ELearningLab.Views;
 
 namespace PowerPointLabs.ELearningLab.Service
 {
+#pragma warning disable 618
```

Comment:
Hi, was just wondering, if you import ActionFramework, do you need this disable line? You can do most things through ActionFramework.

Reply:
I am still getting an error when I use the methods in `ActionFramework`.  I don't understand why this error occurs but adding `#pragma warning disable` seems to suppress the error.